kinto/__main__.py

```diff
@@ -238,11 +231,9 @@ def main(args=None):
         env = bootstrap(config_file, options={"command": "delete-collection"})
         return scripts.delete_collection(env, parsed_args["bucket"], parsed_args["collection"])
 
-    elif which_command == "flush":
-        #env = bootstrap(config_file, option={"command": "flush-cache"})
-        if parsed_args["flush-cache"]:
-            return cache.flush()
-    
+    elif which_command == "flush-cache":
+        return cache.flush()
```

Comment:
Maybe we can add some feedback `cache has been flushed` ?

Reply:
will do :+1: 

also what i dont get yet is how does this work without me referencing to an instance of the cache class,i can't simply call the cache class directly,imo this shouldn't work.

Reply:
What makes you think it does work?

Reply:
yeah that's what i meant it wont work,what instance am i supposed to call here? for it to work

Reply:
Ok I missed that.

If you look at the [migrate command](https://github.com/Kinto/kinto/blob/master/kinto/core/scripts.py#L16-L31) you will get a glance of what do to.

Basically call bootstrap to get the env and then you should be able to call `env["registry"].cache.flush()`

Reply:
I have implemented this,while trying to test it manually using the cli i encountered a bug using make serve

so apparently there is a bug during the installation, doing `make serve` runs it but i did not get the option of choosing the backend,also kinto is not recognized as a command.The troubleshooting doc does not mention anything about fixing this.

Reply:
also the pip file needs to be updated as black=19.xx does not satisfy anymore.